simulation_ws/src/sagemaker_rl_agent/markov/s3_boto_data_store.py

```diff
@@ -98,7 +98,7 @@ def save_to_store(self):
                         s3_client.delete_object(Bucket=self.params.bucket,
                                                 Key=obj["Key"])
                         num_files += 1
-
+                    utils.delete_local_checkpoint(self.params.checkpoint_dir, checkpoint_number_to_delete)
```

Comment:
I think this will only execute if the checkpoint exists in S3. I think we would want to delete the local checkpoints if the exist, even if they aren't in S3. Would it work to move this above the if "Contents" line?